store/tikv/region_cache.go

```diff
@@ -305,6 +309,11 @@ func (c *RegionCache) asyncCheckAndResolveLoop() {
 		case <-c.notifyCheckCh:
 			needCheckStores = needCheckStores[:0]
 			c.checkAndResolve(needCheckStores)
+		case <-ticker.C:
+			// refresh store once a minute to update labels
+			for _, store := range c.storeMu.stores {
```

Comment:
It's reading `c.storeMu.stores` without a read lock. I'm afraid it will cause data races.
Please transform `c.storeMu.stores` to a list in a read lock, then `reResolve` the list one by one.

Reply:
nice catch, updated.